broker/broker.go

```diff
@@ -219,3 +228,33 @@ func copyDependencies(workDir string) error {
 
 	return nil
 }
+
+func printDebug(format string, v ...interface{}) {
+	log.Printf(format, v...)
+}
+
+func printInfo(format string, v ...interface{}) {
+	message := fmt.Sprintf(format, v...)
+	bus.Pub(&model.PluginBrokerLogEvent{
+		RuntimeID: cfg.RuntimeID,
+		Text:      message,
+		Time:      time.Now(),
+		Stream:    model.StdoutStream,
+	})
+}
+
+func printError(format string, v ...interface{}) {
+	message := fmt.Sprintf(format, v...)
+	log.Fatal(message)
+}
+
+func printPlan(metas []model.PluginMeta) {
+	var buffer bytes.Buffer
+
+	buffer.WriteString("Planning to install\n")
```

Comment:
s/Planning to install/List of plugins and editors to install/

Reply:
Thanks. Fixed.